plugins/modules/monitoring/newrelic_deployment.py

```diff
@@ -100,11 +105,11 @@ def main():
             application_id=dict(required=False),
             changelog=dict(required=False),
             description=dict(required=False),
-            revision=dict(required=False),
+            revision=dict(required=True),
             user=dict(required=False),
             appname=dict(required=False),
             environment=dict(required=False),
-            validate_certs=dict(default=True, type='bool'),
+            validate_certs=dict(required=False, default=True, type='bool'),
```

Comment:
Why is the certificates' validation being ditched? DNS hijacking could be mitigated by validating the certificate, no?

Reply:
Thanks @russoz for taking the time to review. For validate_certs, tbh I thought it was from the v1 API, will get that reverted.